diffusion/models/models.py

```diff
@@ -219,8 +222,10 @@ def stable_diffusion_xl(
         unet = UNet2DConditionModel.from_pretrained(unet_model_name, subfolder='unet')
     else:
         config = PretrainedConfig.get_config_dict(unet_model_name, subfolder='unet')
+        if use_e5:
+            # e5 + clip embedding dims + micro conditioning
+            config[0]['cross_attention_dim'] = 2304
```

Comment:
Should we just make this configurable as a seperate arg instead of having it overridden with a boolean?

Reply:
I'm going to leave this as is to avoid the mental burden of a user having know or figure out what the xattention dim should be if they enable e5. It's ugly, and something that would ideally be enabled via it's own unique HF checkpoint but mocking said behavior like this is probably the best we can do atm.